src/frontend/src/lib/constants/app.constants.ts

```diff
@@ -43,6 +43,12 @@ export const BACKEND_CANISTER_ID = LOCAL
 		? import.meta.env.VITE_STAGING_BACKEND_CANISTER_ID
 		: import.meta.env.VITE_IC_BACKEND_CANISTER_ID;
 
+export const SIGNER_CANISTER_ID = LOCAL
+	? import.meta.env.VITE_LOCAL_SIGNER_CANISTER_ID
+	: STAGING
+		? import.meta.env.VITE_STAGING_SIGNER_CANISTER_ID
+		: import.meta.env.VITE_IC_SIGNER_CANISTER_ID;
```

Comment:
The logic for getting the signer canister ID is identical to that for the backend.  I guess it might diverge in future though.